frontend/libs/erxes-ui/src/modules/inputs/components/PhoneField.tsx

```
@@ -319,6 +337,172 @@ const PhoneForm = () => {
+    },
+    ref,
+  ) => {
+    const setPhones = useSetAtom(phonesFamilyState(recordId));
+    const setShowPhoneInput = useSetAtom(showPhoneInputFamilyState(recordId));
+
+    useEffect(() => {
+      setPhones(phones);
+      return () => {
+        setShowPhoneInput(false);
+      };
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)